Source/ChocolateyGui.Subprocess/packages.config

```diff
@@ -1,34 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="AlphaFS" version="2.0.1" targetFramework="net452" />
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
-  <package id="Castle.Core" version="4.0.0" targetFramework="net452" />
   <package id="chocolatey.lib" version="0.10.5" targetFramework="net452" />
-  <package id="Fleck" version="0.13.0.57" targetFramework="net452" />
   <package id="log4net" version="2.0.3" targetFramework="net452" />
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net452" />
-  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net452" />
-  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net452" />
-  <package id="MsgPack.Cli" version="0.8.1" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net452" />
-  <package id="Newtonsoft.Msgpack" version="0.1.10" targetFramework="net452" />
-  <package id="Nito.AsyncEx" version="4.0.1" targetFramework="net452" />
+  <package id="Microsoft.VisualStudio.Threading" version="15.3.20" targetFramework="net452" />
+  <package id="Microsoft.VisualStudio.Validation" version="15.3.15" targetFramework="net452" />
   <package id="Serilog" version="2.4.0" targetFramework="net452" />
   <package id="Serilog.Sinks.Async" version="1.0.1" targetFramework="net452" />
   <package id="Serilog.Sinks.File" version="3.2.0" targetFramework="net452" />
   <package id="Serilog.Sinks.Literate" version="2.1.0" targetFramework="net452" />
   <package id="Serilog.Sinks.RollingFile" version="3.3.0" targetFramework="net452" />
   <package id="StyleCop.Analyzers" version="1.0.0" targetFramework="net452" developmentDependency="true" />
-  <package id="System.Collections.Immutable" version="1.3.1" targetFramework="net452" />
-  <package id="System.Reactive" version="3.1.1" targetFramework="net452" />
   <package id="System.Reactive.Core" version="3.1.1" targetFramework="net452" />
   <package id="System.Reactive.Interfaces" version="3.1.1" targetFramework="net452" />
   <package id="System.Reactive.Linq" version="3.1.1" targetFramework="net452" />
-  <package id="System.Reactive.PlatformServices" version="3.1.1" targetFramework="net452" />
-  <package id="System.Reactive.Windows.Threading" version="3.1.1" targetFramework="net452" />
-  <package id="System.Threading.Tasks.Dataflow" version="4.7.0" targetFramework="net452" />
-  <package id="System.ValueTuple" version="4.3.1" targetFramework="net452" />
-  <package id="WampSharp" version="1.2.5.27-beta" targetFramework="net452" />
-  <package id="WampSharp.Fleck" version="1.2.5.27-beta" targetFramework="net452" />
-  <package id="WampSharp.NewtonsoftJson" version="1.2.5.27-beta" targetFramework="net452" />

```

Comment:
That was 90% of the reason for this change. WampSharp just had some many superflous dependencies and issues.